src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__box {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__panel {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  position: relative;
+}
+
+.list__item {
+  margin-left: 20px;
+}
+
+.list__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  font-family: Roboto, sans-serif;
```

Comment:
Font should be set just for the whole page

Reply:
Corrected